package.json

```diff
@@ -47,12 +47,12 @@
     "parseurl": "~1.3.3",
     "path-to-regexp": "0.1.7",
     "proxy-addr": "~2.0.5",
-    "qs": "6.7.0",
+    "qs": "6.9.1",
```

Comment:
This should not be updates if it causes a dup in the tree. It is updated along with at least body-parser typically to keep it from getting duplicated.

Reply:
Thanks for the explanation! 👍